irods/manager/access_manager.py

```diff
@@ -104,7 +104,8 @@ def __get_raw(self, target, **kw):
         else:
             raise TypeError
 
-        rows  = [ r for r in query_func(target.path) ]
+        extant_ids = set(u[User.id] for u in self.sess.query(User))
+        rows  = [r for r in query_func(target.path) if r[access_column.user_id] in extant_ids]
```

Comment:
Should we add a TODO and link to irods/irods#6921 here, too?

Reply:
What is the task for the TODO you mention -- does it concern this issue/PR, or more irods#6921?

Reply:
More irods/irods#6921. I imagine we won't need to run the extra query in the future once that issue is resolved, right? I could be missing the point here.

Reply:
Right, I see.

Reply:
Don't know yet what a solution to https://github.com/irods/irods/issues/6921 will guarantee, but if a user/group removal includes the removal of the ID from `r_objt_access` table within the transaction, maybe the extra query can be removed.

Reply:
irods/irods#6921 linked now in comments , commit message, and PR title.

Reply:
Issue now linked in comment, title and commit msg